ui/page.go

```diff
@@ -204,26 +203,17 @@ func (page pageCommon) LayoutWithAccounts(gtx *layout.Context, body layout.Widge
 			Label: page.theme.Body1(page.info.Wallets[*page.selectedWallet].Accounts[i].Name),
 		}
 	}
+
+	title := page.theme.Label(values.TextSize18, "Accounts:")
+	page.accountsTab.SetTitle(&title)
```

Comment:
There's no need to pass a pointer to SetTitle. If you want to check whether the tab has a title label, you can check with Label.shaper.